api/envoy/config/filter/http/ext_authz/v2alpha/ext_authz.proto

```diff
@@ -68,23 +77,52 @@ message HttpService {
   // Sets the HTTP server URI which the authorization requests must be sent to.
   envoy.api.v2.core.HttpUri server_uri = 1;
 
-  // Sets an optional prefix to the value of authorization request header *Path*.
-  string path_prefix = 2;
```

Comment:
Do we need to go through a deprecation cycle here? I'm unclear of who is using this filter currently in production?

Reply:
Good point, I thought it was ok to do breaking changes in alpha. Let me look into this again.

Reply:
Are we sure about this? I agree that because it's alpha we can technically change it, but I get the impression that a bunch of people are now using this filter. After this change, if we decide to do it this way, I think we should actually switch this away from alpha in a subsequent PR? Thoughts? Can we track switching this away from alpha in a follow up issue?

Reply:
@mattklein123 Sorry, just saw this comment. Yes, I think after this change we could move it away from alpha. I can track it in a follow-up PR, but there are couple issues that I think I will need to address before this happens. One issue is that the HTTP client crashes envoy on the first request if cluster name doesn't correspond to any actual cluster. The other issue is this https://github.com/envoyproxy/envoy/issues/4637.